doc/guides/SimpleStartupGuide.md

```diff
@@ -422,7 +449,9 @@ There are multiple commands that can update aspects of the camera entry in meta-
 
 ## Execute GetStreamURI Command through EdgeX
 
-1. Get the profile token by executing the `GetProfiles` command:
+1. <a name="step1"></a>Get the profile token by executing the `GetProfiles` command:
+
+   >NOTE: Make sure to replace `Camera001` in all the commands below, with the deviceName returned in the ["Verify device(s) have been successfully added to core-metadata"](#verify-device) step above.  
```

Comment:
i dont think the verify-device link works

Reply:
would you happen to know how to create an anchor to something other than a heading?

Reply:
This appears to work in github